debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java

```diff
@@ -1482,25 +1482,25 @@ public static <T> T querySingleValue(Connection connection, String queryString,
         }
     }
 
-    public String buildSelectWithRowLimits(TableId tableId, int limit, String projection, Optional<String> condition,
-                                           Optional<String> additionalCondition, String orderBy) {
+    public String buildSelectWithRowLimits(TableId tableId, int limit, String projection, String condition,
```

Comment:
@mfvitale Why is this change needed?

Reply:
Just removed the `Optional` as parameters since it's not a good practice. Now, in case of missing value is simply empty `""`

Reply:
I don't think I agree on that one. We use it in the codebase elsewhere as well. The `Optional` is a way how to conway the message in code that value might be present. Even if originally it was told that Optional should be mostly return value there are differing opinions on that rule too.

Reply:
The main "issue" (it's more about clean code) with Optional in the parameter is that it causes conditional logic inside methods, like passing boolean. This means that the method will have two different flow inside and so doing to much things. Leveraging on  method overloading or method with different name can be better. 

But it is just a different opinion on that. We can discuss it apart.